tests/test_fixtures.py

```diff
@@ -0,0 +1,172 @@
+"""Ensure the test fixtures work as expected."""
+from typing import cast
+
+import pytest
+from bs4 import BeautifulSoup
+from playwright.sync_api import Page
+from playwright.sync_api import Route
+from starlette.testclient import TestClient
+
+from psc.fixtures import DummyPage
+from psc.fixtures import DummyRequest
+from psc.fixtures import DummyResponse
+from psc.fixtures import DummyRoute
+from psc.fixtures import FakeDocument
+from psc.fixtures import MockTestClient
+from psc.fixtures import PageT
+from psc.fixtures import mocked_client_page
+from psc.fixtures import route_handler
+from psc.here import STATIC
+
+
+Element = cast("Unknown", "Element")
```

Comment:
The addition of this line causes some tests to fail, as `Element` should only exist in tests that ask for the `fake_element` fixture.